### ISSUE TITLE
PLE1206: false-positive for star args

### ISSUE THREAD

**Issue by spaceone** [charliermarsh/ruff#3255]
```python
data = foo, bar, baz
logger.debug("foo=%r, bar=%s, baz=%r", *data)
```
is detected as PLE1206 logging-too-few-args

**Comment by charliermarsh** [comment]
Should already be fixed on `main` (just tested it).
